src/woo_search/search_index/tests/test_search_api.py

```diff
@@ -1432,7 +1572,7 @@ def test_filter_by_publisher_and_information_category_and_result_type(self):
             {
                 "publishers": ["f9cc8c26-7ce7-4a25-9554-e6a2892176d7"],
                 "informatieCategorieen": ["c9001845-aef0-4150-bbf0-a5f5c096e603"],
-                "resultTypes": [ResultTypeChoices.document],
+                "resultTypes": [ResultTypeChoices.publication],
```

Comment:
this changes the whole semantics of the test - can't you add a topic to one of the records that hit instead?